lib/src/realm_class.dart

```diff
@@ -62,6 +63,11 @@ class Realm {
     }
   }
 
+  static void deleteRealm(Configuration config) {
```

Comment:
As the comment on `Realm.exists` I think we should require a path here. Please check my reasoning [here ](https://github.com/realm/realm-dart/pull/171#discussion_r783772877)

Reply:
@nielsenko @desistefanova ^

Reply:
Fixed in https://github.com/realm/realm-dart/pull/169/commits/0697dce4c21edbd697003715971e4e21b9d68d23